scripts/ops/sentry-agent-loop.mjs

```
@@ -0,0 +1,1530 @@
+      ['worktree', 'remove', '--force', worktree],
+      ['branch', '-D', branch],
+    ]) {
+      const result = await run('git', args, { cwd: root, env: runtimeEnv });
+      if (result.status !== 0) {
+        cleanupFailures.push(commandFailure('git', args, result).message);
+      }
+    }
+    if (cleanupFailures.length > 0) {
+      throw new Error(
+        `failed to remove disposable Git worktree and branch:\n${cleanupFailures.join('\n')}`
+      );
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)